deploy/infra/lib/constructs/app-backend.ts

```diff
@@ -310,4 +320,8 @@ export class AppBackend extends Construct {
   getIdpSync(): IdpSync {
     return this._idpSync;
   }
+
+  getKMSKey(): string {
```

Comment:
Lets call this getKmsKeyArn
```suggestion
  getKmsKeyArn(): string {
```